docs/book/known_issues.md

```diff
@@ -75,3 +78,23 @@ Issue 9<a id="issue_9"></a>: CnsRegisterVolume API does not validate if the volu
 
 - Impact: One of the PVC using the CNS volume will be usable at any point in time. Usage of any other PVC will lead to attach failures.
 - Workaround: None
+
+Issue 10<a id="issue_10"></a>: The Supervisor Cluster PVC expanded size is lost and cannot be reclaimed in vSphere 7.0U1 patch release.
```

Comment:
7.0 U1 is not a patch release. Could you remove "patch" from here and above? You may want to say vSphere 7.0 Update 1

Reply:
Sure. I will remove it.